editor/plugins/script_text_editor.cpp

```diff
@@ -941,10 +941,17 @@ void ScriptTextEditor::_edit_option(int p_op) {
 				if (tx->get_selection_to_column() == 0)
 					end -= 1;
 
+				// Check if all lines in the selected block are commented
+				bool  is_commented = true;
+				for (int i = begin; i <= end; i++) {
+					if (!tx->get_line(i).strip_edges().begins_with("#"))
+						is_commented = false;
+						break;
```

Comment:
This will always break on the first loop.

Reply:
Yes, please brace all your `if`s. I know some older code doesn't, but it's a good practice to do so in new code.

Reply:
thanks, python habits, sry. Waiting for godot to build then upload changes.